src/Plugins/Loader/ManagedLoadContext.cs

```diff
@@ -157,6 +157,19 @@ private bool SearchForLibrary(ManagedLibrary library, out string path)
                 }
             }
 
+            // 3. Search in _basePath/runtimes
```

Comment:
Can you explain more for me when this is used and provide examples? In this code, I'm attempting to imitate what corehost does, and although I don't know everything about corehost's inner workings, I'm not familiar with a loading convention like this.

Reply:
If I run without my modifications to your project then my msbuild PowerShellTaskFactory will fail to load with, for example:
````
Exception thrown: 'System.IO.FileNotFoundException' in Unknown Module.
Could not load file or assembly 'System.Management.Automation, Version=6.0.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. occurred
````
This is in spite of the fact that System.Management.Automation is present in my plugins publish folder at these locations:

```
publish\runtimes\linux-arm\lib\netstandard2.0\System.Management.Automation.dll
publish\runtimes\linux-x64\lib\netstandard2.0\System.Management.Automation.dll
publish\runtimes\osx\lib\netstandard2.0\System.Management.Automation.dll
publish\runtimes\win-arm\lib\netstandard2.0\System.Management.Automation.dll
publish\runtimes\win-arm64\lib\netstandard2.0\System.Management.Automation.dll
publish\runtimes\win-x64\lib\netstandard2.0\System.Management.Automation.dll
publish\runtimes\win-x86\lib\netstandard2.0\System.Management.Automation.dll
```
...and in spite of the fact that at the time that ManagedLoadcontext.SearchForLibrary() gives up looking for the assembly and returns null, that the value of library.AdditionalProbingPath is:

    system.management.automation\6.0.4\runtimes/win-x64/lib/netstandard2.0/System.Management.Automation.dll

I surmised that your loader had kindly set up the library object with some sort of nuget cache path which was tantalizing close to the path required to find it in my plugins own deployment (publish directory). All that was needed was to tear off the \<packagename>/\<packageversion> at the start of the above relative path and then append that to _basepath. So my change does just that and then my PowerShellTaskFactory works (inside of "dotnet msbuild" on Windows and Mac where I have tested it anyway).